pkg/storage/stores/shipper/compactor/deletion/validation_test.go

```diff
@@ -8,32 +8,34 @@ import (
 
 func TestParseLogQLExpressionForDeletion(t *testing.T) {
 	t.Run("invalid logql", func(t *testing.T) {
-		matchers, err := parseDeletionQuery("gjgjg ggj")
-		require.Nil(t, matchers)
+		logSelectorExpr, err := parseDeletionQuery("gjgjg ggj")
+		require.Nil(t, logSelectorExpr)
 		require.ErrorIs(t, err, errInvalidQuery)
 	})
 
 	t.Run("matcher expression", func(t *testing.T) {
-		matchers, err := parseDeletionQuery(`{env="dev", secret="true"}`)
-		require.NotNil(t, matchers)
+		logSelectorExpr, err := parseDeletionQuery(`{env="dev", secret="true"}`)
+		require.NotNil(t, logSelectorExpr)
 		require.NoError(t, err)
 	})
 
 	t.Run("pipeline expression with line filter", func(t *testing.T) {
-		matchers, err := parseDeletionQuery(`{env="dev", secret="true"} |= "social sec number"`)
-		require.Nil(t, matchers)
-		require.ErrorIs(t, err, errUnsupportedQuery)
+		logSelectorExpr, err := parseDeletionQuery(`{env="dev", secret="true"} |= "social sec number"`)
+		require.NotNil(t, logSelectorExpr)
+		require.NoError(t, err)
 	})
 
+	/* syntax.ParseLogSelector does not reject these
```

Comment:
Why are these commented out? If they're not useful we should probably delete them